src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -429,7 +421,16 @@ def load_categorical_paf(key: str, location: str) -> pd.DataFrame:
         )
 
     exp = get_data(risk.EXPOSURE, location)
-    rr = get_data(risk.RELATIVE_RISK, location)
+
+    if key == data_keys.STUNTING.PAF or key == data_keys.WASTING.PAF:
+        national_location_id = get_national_location_id(location[0])
```

Comment:
The typing on the function `load_categorical_paf` suggests that `location` is a string. If this is correct, `location[0]` will get the first character of the location. If `location` is actually a list of strings, the typing of this method should be updated to reflect this.

This appears to be the case in several other methods in this file as well.

Reply:
@SylLutze It will be `location: Union[str, List[int]]`

Reply:
I had left them as strings for national data keys methods but we are now making some of them to pull subnational data.